cainafrica/src/components/partners-sponsors.js

```diff
@@ -0,0 +1,138 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Actually, these list of partners have not provided permission for CAIN to use their logos. Please remove this page for now so I can approve this request. then when their permission has been provided, you can add it back. thanks